src/test/java/DataFetcherWithErrorsTest.java

```diff
@@ -0,0 +1,99 @@
+import graphql.ErrorType;
+import graphql.ExecutionResult;
+import graphql.GraphQL;
+import graphql.GraphQLError;
+import graphql.execution.DataFetcherResult;
+import graphql.language.SourceLocation;
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import graphql.schema.GraphQLSchema;
+import graphql.schema.StaticDataFetcher;
+import graphql.schema.idl.RuntimeWiring;
+import graphql.schema.idl.SchemaGenerator;
+import graphql.schema.idl.SchemaParser;
+import graphql.schema.idl.TypeDefinitionRegistry;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static graphql.schema.idl.RuntimeWiring.newRuntimeWiring;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.emptyMap;
+import static java.util.Collections.singletonList;
+import static java.util.Collections.singletonMap;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+
+public class DataFetcherWithErrorsTest {
```

Comment:
I know java tests aren't what this project uses, but I wanted to get something out and show how it works before trying to figure out how the groovy test stuff works.

Reply:
yes, that should be Spock before it will be merged.

Reply:
fixed